integration-tests/src/test/resources/DomainOnPVUsingWLST.yaml

```diff
@@ -2,7 +2,7 @@
 # Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
 
 adminServerName: admin-server
-domainUID: domain1
+domainUID: domainonpvwlst
```

Comment:
Can we have the file name match the domainUID as a convention?  That would simplify correlating them, and keep the filenames short.   

Also, can we have a comment at the beginning of each domain setup file that explains its topology?   Ideally, we should be able to easily determine what makes the domain unique by looking at the first few lines of each domain file...

Similarly, it'd be helpful if every operator setup file matched its namespace name.

Reply:
Ok. Are you saying operator setup file name should have the namespace where operator is deployed?

Reply:
@tbarnes-us what should be the operator file name for OperatorManagingdefaultAndtest1NS.yaml based on your suggestion?

Reply:
It can be the same as the operator's namespace, which I assume is unique...

Reply:
ok, changes made.